Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "griddle"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
I believe this is a breaking change, as from memory we expose some of `hashbrown`'s types (and `ahash`'s type) directly in the `griddle` API, so any major version bump to those also requires us to issue a breaking release. Just remove this for now :)

Reply:
Has been restored in the latest submission.